cmd/tools/cli/cli.go

```diff
@@ -23,10 +23,14 @@ var rootCmd = &cobra.Command{
 	SilenceErrors: true,
 }
 
-var pluginRepoURI string
+var (
+	pluginRepoURI string
+	useK3s        bool
+)
 
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&pluginRepoURI, "plugin-repo", "", plugins.DefaultPluginURI, "URI for the plugins repository. Must begin with https:// or file://")
+	rootCmd.PersistentFlags().BoolVarP(&useK3s, "k3s", "", true, "Use k3s for deployment. Uses kubectl when set to false")
```

Comment:
Since it is a boolean, I was thinking more `--use-kubectlt` without set `=true/false`. 
or maybe easier `--kubectl`  

so instead of having:
```
bin/k3ai-cli delete kubeflow-pipelines --k3s=false
```
it will be:
```
 bin/k3ai-cli delete kubeflow-pipelines --kubectl
```

Reply:
I like the proposal of not setting true/false. I'd vote for `--use-kubectlt` as it's self-explanatory.

Reply:
Makes sense.